### PR TITLE
Rename profile page to details page

### PR DESCRIPTION
**Pull request by NikkiWines**
### Details
Currently in E.cash there are two profile pages: The page you get when you click on the chat header avatar of a chat you have with someone, and the page you get when navigating to Settings > Profile. 

The non-settings profile page displays a header of `Details` while the settings profile page displays `Profile`. 

There's no reason for these pages to be named the same thing, and from the description above you can see how much more difficult to describe which page is which. 

### Fixed Issues
Fixes N/A, was brought up in conversation and is an easy fix. 

### Tests
1. Log into E.cash 
2. Create a chat with one person 
3. Click on their chat header avatar and confirm you can see the `Details` page for that user. 

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web
Screen Shot 2021-03-19 at 10 59 21 AM

#### Desktop
Screen Shot 2021-03-19 at 11 00 15 AM

### CODE REVIEW

File: src/pages/home/HeaderView.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ const HeaderView = props => (
                         onPress={() => {
                             const {participants} = props.report;
                             if (participants.length === 1) {
-                                Navigation.navigate(ROUTES.getProfileRoute(participants[0]));
```
Comment (jasperhuangg) [inline src/pages/home/HeaderView.js:66]: I know this is kinda unrelated to your changes, but it looks like `participants` is a duplicate name from the upper scope (line 53). Is there any reason we're re-accessing/redefining a variable for `props.reports.participants` here?

**Review by jasperhuangg — CHANGES_REQUESTED** [review]
@NikkiWines Looks good! Merge conflicts just need to be solved and then it should be good to go!

File: src/pages/home/HeaderView.js (hunk 1 above)
Comment (NikkiWines) [reply src/pages/home/HeaderView.js]: You're right that there's no need to redefine this here and actually it convenient got changed/resolved after merging master.

**Comment by NikkiWines** [comment]
Updated and resolved conflicts

**Review by jasperhuangg — APPROVED** [review]

**Pull request merged**